source/common/res/features/popup-calculator/main.css

```diff
@@ -1,13 +1,14 @@
-#toolkitPopupCalc {
+#xtoolkitPopupCalc {
```

Comment:
Yeah, the x version should be deleted as that class is not needed in that css file.